docsite/rst/ansible_ssh_changes_note.rst

```diff
@@ -1,3 +1,4 @@
 .. note::
 
     Ansible 2.0 has deprecated the “ssh” from ``ansible_ssh_user``, ``ansible_ssh_host``, and ``ansible_ssh_port`` to become ``ansible_user``, ``ansible_host``, and ``ansible_port``. If you are using a version of Ansible prior to 2.0,  you should continue using the older style variables (``ansible_ssh_*``). These shorter variables are ignored, without warning, in older versions of Ansible. 
+    Please pay attention to the older style variables (``ansible_ssh_*``): ansible_ssh_password doesn't exist, should be ansible_ssh_pass. 
```

Comment:
I'd like to suggest a bit of editing here... maybe this instead:

For older variable styles, please also note that ``ansible_ssh_password`` does not exist and should be entered as ``ansible_ssh_pass`` instead.